doc/source/serve/doc_code/monitoring/monitor_deployment.py

```diff
@@ -0,0 +1,34 @@
+# flake8: noqa
+# fmt: off
+
+# __monitor_start__
+import dataclasses
+from typing import List
+
+from ray import serve
+from ray.serve.schema import ServeStatus
+
+
+@serve.deployment
+def get_healthy_apps() -> List[str]:
+    serve_status: ServeStatus = serve.status()
+    app_statuses = dataclasses.asdict(serve_status)["applications"]
```

Comment:
Should we just access the dataclass attributes directly instead of converting to a dict?

Reply:
Good idea, I changed it to use dataclass attributes directly.